### DISCUSSION TITLE
Guide to send alerts for ps5 & xbox to their own individual discord channels?

### DISCUSSION THREAD

**Discussion by lozbinks** [jef/streetmerchant#1938] [Help]
So I have created a discord server with two seperate channels (xbox-x & ps5) I'm very much of a discord novice btw but trying to learn.

I have tried creating webhooks for each channel and added them to the dotenv seperated by a comma as below

DISCORD_WEB_HOOK="https://discord.com/api/webhooks/xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx", "https://discord.com/api/webhooks/zzzzzzzzzzzzzzzzzzzzzzzzzzzzz"

I had the notify group settings set up to their corresponding channel name as below 

DISCORD_NOTIFY_GROUP_XBOXSX= "@xbox-x Xbox Series X is in stock!"

and 

DISCORD_NOTIFY_GROUP_SONYPS5C="@ps5 Playstation 5 is in stock! "

However it only seemed to use the first webhook and posted both series into one channel not their corresponding one as intended.

I did read another post on here #1374 but unfortunate to say I'm still at a loss and currently now just have one webhook and both series going to one channel.

I'm pretty sure there is something very simple that I'm doing wrong and would appreciate if anyone could point out what that may be.

**Comment by OptimalMising** [comment]
Out of the box streetmerchant can only post to one discord channel and the Notify groups are for server roles to do @mentions (not for channels).

So the easiest approach to work with the OOTB functionality is to:
* Create a singular discord channel eg #alerts_consoles (get the webhook for this and add it to the dot env file)
* Set up two roles on your discord sever, one for PS5 one for Xbox Series X.  Follow the instructions to add your role id's to the dotenv environment file.

When any stock is found (ps5's for example) the notification will @ mention the server role so all those subscribed or assigned (for ps5's) will get pinged about the notification.
